docs/build/limits.md

```diff
@@ -2,14 +2,30 @@
 title: Limits
 ---
 
-OpenFn has a number of configurable limits that help ensure smooth operation.
-This page details what those limits are and how they can be adjusted.
+OpenFn cloud hosted instance has a number of limits that help ensure smooth
+operation. This page details what those limits are and how they can be adjusted.
+
+| Feature                     | Description                                                  | DPG          | Free    | Core   | Growth  | Scale     | Unlimited |
+| --------------------------- | ------------------------------------------------------------ | ------------ | ------- | ------ | ------- | --------- | --------- |
+| Runs                        | Maximum number of runs allowed per month                     | Unlimited    | 100     | 2,000  | 5,000   | 10,000    | Unlimited |
+| Workflow Execution Duration | Maximum time a workflow can run before being killed          | Configurable | 60 secs | 5 mins | 20 mins | 30 mins   | 30 mins   |
+| Memory Usage                | Maximum memory allowed per workflow attempt                  | Configurable | 128MB   | 256MB  | 512GB   | 1GB       | 1GB       |
+| Dataclip Size               | Maximum size for data clips persisted run states             | Configurable | 512KB   | 2MB    | 10MB    | 10MB      | 10MB      |
+| AI Assistant                | Maximum AI tokens available                                  | Configurable | 500K    | 1.5M   | 5M      | 10M       | 10M       |
+| Data Collections (Storage)  | Maximum storage for data collections                         | Configurable | 1MB     | 5MB    | 10MB    | 50MB      | 50MB      |
+| Data Collections (Number)   | Maximum number of data collections per project               | Configurable | 2       | 5      | 10      | Unlimited | Unlimited |
+| Concurrency Control         | Allowing users to control concurrency limits for the project | Configurable | Yes     | Yes    | Yes     | Yes       | Yes       |
+| Workflow concurrency limit  | Maximum workflow runs that can be executed in parallel       | Configurable | N/A     | N/A    | N/A     | N/A       | N/A       |
```

Comment:
@christad92 what do you mean by 'N/A' here? I think these concurrency features are available for all plans. Maybe we say something like 'configurable' rather?

Reply:
We currently don't have limits for workflow concurrency. I think this is something we will introduce sometime soon but in the mean time.. it is Not Applicable hence the N/A.